Gemfile

```diff
@@ -3,71 +3,24 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
```

Comment:
Cleaned up the gemfile and removed unused gems for both readability and app "weight".